src/containers/sprite-library.jsx

```diff
@@ -11,18 +11,55 @@ class SpriteLibrary extends React.Component {
     constructor (props) {
         super(props);
         bindAll(this, [
-            'handleItemSelect'
+            'handleItemSelect',
+            'handleMouseEnter',
+            'handleMouseLeave',
+            'rotateCostume'
         ]);
+        this.state = {
+            activeSprite: null,
+            costumeIndex: 0,
+            sprites: spriteLibraryContent
+        };
+    }
+    componentWillReceiveProps (newProps) {
+        if (!newProps.visible) clearInterval(this.intervalId);
     }
     handleItemSelect (item) {
         this.props.vm.addSprite2(JSON.stringify(item.json));
     }
+    handleMouseEnter (item) {
+        this.setState({activeSprite: item});
+        if (this.intervalId) clearInterval(this.intervalId);
+        this.intervalId = setInterval(this.rotateCostume, 300);
```

Comment:
Since `this.rotateCostume` needs `this.state.activeSprite`, I think it would be more correct to set the interval in the callback for `setState`, so that `activeSprite` is ensured to be there.